### ISSUE TITLE
After game start, game panel does not have keyboard focus.

### ISSUE THREAD

**Issue by andreas-eberle** [jsettlers/settlers-remake#413]
After you started a game, you can only use keybindings after you clicked into the games panel at least once. However, this should work from the start.
